client/state/action-types.js

```diff
@@ -352,6 +352,7 @@ export const INVITES_REQUEST_SUCCESS = 'INVITES_REQUEST_SUCCESS';
 export const INVITE_RESEND_REQUEST = 'INVITE_RESEND_REQUEST';
 export const INVITE_RESEND_REQUEST_FAILURE = 'INVITE_RESEND_REQUEST_FAILURE';
 export const INVITE_RESEND_REQUEST_SUCCESS = 'INVITE_RESEND_REQUEST_SUCCESS';
+export const JETPACK_CONNECTION_CHANGE_OWNER = 'JETPACK_CONNECTION_CHANGE_OWNER';
```

Comment:
It's worth mentioning that we currently store the corresponding WP.com user data of the connected user in the `state.jetpack.connection.dataItems` reducer, and, and that reducer is currently being manipulated by the `JETPACK_USER_CONNECTION_DATA_RECEIVE` action type. The `JETPACK_CONNECTION_*` action types are currently used for the generic connection information (whether site is active, whether it is on staging, whether dev mode is active), which isn't as related to the connection ownership. Do you think it would make sense to go with a consistent naming here, and use `JETPACK_USER_CONNECTION_CHANGE_OWNER` for example? Or, if necessary, update the current `JETPACK_USER_CONNECTION_DATA_RECEIVE` to something else that will accommodate the consistency?

Reply:
>  Do you think it would make sense to go with a consistent naming here, and use JETPACK_USER_CONNECTION_CHANGE_OWNER for example?

Aha, I did not realise this. Yes, let's change to `JETPACK_USER_CONNECTION_CHANGE_OWNER`.